Source/WebKit/Shared/RemoteLayerTree/RemoteLayerBackingStore.h

```diff
@@ -91,6 +93,10 @@ class RemoteLayerBackingStore : public CanMakeWeakPtr<RemoteLayerBackingStore> {
     virtual bool isRemoteLayerWithRemoteRenderingBackingStore() const { return false; }
     virtual bool isRemoteLayerWithInProcessRenderingBackingStore() const { return false; }
 
+    enum class ProcessModel : uint8_t { InProcess, Remote };
+    virtual ProcessModel processModel() const = 0;
```

Comment:
This is a little bit confusing. The class name is `RemoteLayerBackingStore` and it can be `InProcess`. I would expect it should always be `Remote`. But it looks like the meaning of "remote" in the name is different from the meaning of "remote" in the `ProcessModel`

Reply:
Oh I see now. The `ProcessModel` is related to the `RenderingBackingStore` if it is in-process or remote. Maybe including this in the name would make it clearer.

Reply:
We have a few too many "Remotes" these days :)